src/shared/modules.mjs

```diff
@@ -854,6 +854,167 @@ const generateEventListenResponse = json => {
     return json
 }
 
+const getRefDefinition = (uri = '', moduleJson = {}) => {
```

Comment:
I think we already have a method in json-schema.json that does this. it's fine for modules.mjs to depend on json-schema, since all modules are json-schema files.

Can you try and leverage the existing methods for this? I think it's `getPath`

Reply:
Let me check and try

Reply:
import { getPath as getRefDefinition } from './json-schema.mjs', since getPath is already importing from crocks/Maybe/getPath.js too